### PR TITLE
GG-26737 Updated dependencies

### PR DESCRIPTION
**Pull request by akuznetsov-gridgain**

### CODE REVIEW

File: modules/zookeeper/pom.xml [hunk 1]
```diff
@@ -40,6 +40,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
```
Comment (nva) [inline modules/zookeeper/pom.xml]: Please revert this. This changes doesn't related with your ticket

File: modules/zookeeper/src/main/java/org/apache/ignite/spi/discovery/tcp/ipfinder/zk/TcpDiscoveryZookeeperIpFinder.java [hunk 2]
```diff
@@ -42,7 +42,7 @@
 import org.apache.ignite.resources.LoggerResource;
 import org.apache.ignite.spi.IgniteSpiException;
 import org.apache.ignite.spi.discovery.tcp.ipfinder.TcpDiscoveryIpFinderAdapter;
-import org.codehaus.jackson.map.annotate.JsonRootName;
+import com.fasterxml.jackson.annotation.JsonRootName;
```
Comment (nva) [inline modules/zookeeper/src/main/java/org/apache/ignite/spi/discovery/tcp/ipfinder/zk/TcpDiscoveryZookeeperIpFinder.java]: Please revert this. This changes doesn't related with your ticket

File: parent/pom.xml [hunk 3]
```diff
@@ -76,8 +76,7 @@
         <hamcrest.version>1.2</hamcrest.version>
         <httpclient.version>4.5.1</httpclient.version>
         <httpcore.version>4.4.3</httpcore.version>
-        <jackson.version>2.9.9</jackson.version>
-        <jackson1.version>1.9.13</jackson1.version>
```
Comment (nva) [inline parent/pom.xml]: Please revert this. This changes doesn't related with your ticket

File: modules/zookeeper/pom.xml (hunk 1 above)
Comment (akuznetsov-gridgain) [reply modules/zookeeper/pom.xml]: Reverted

File: modules/zookeeper/src/main/java/org/apache/ignite/spi/discovery/tcp/ipfinder/zk/TcpDiscoveryZookeeperIpFinder.java (hunk 2 above)
Comment (akuznetsov-gridgain) [reply modules/zookeeper/src/main/java/org/apache/ignite/spi/discovery/tcp/ipfinder/zk/TcpDiscoveryZookeeperIpFinder.java]: Reverted

File: parent/pom.xml (hunk 3 above)
Comment (akuznetsov-gridgain) [reply parent/pom.xml]: Reverted

**Pull request merged**